pkg/errors/custom_errors.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
```

Comment:
Please move this file to the `kubevirt` package, the errors declared here are specific to `kubevirt`.

Reply:
Does it make sense to leave it here, so if any new custom errors will be added here(just like the cloud provider mc) ?

Reply:
After a second thought, moving to the package kubevirt is better.